src/main/java/com/bankingservice/banking/controller/AccountController.java

```diff
@@ -78,6 +80,6 @@ public BaseResponseDTO healthCheck(){
      */
 
     private MetaDTO createMetaData(){
```

Comment:
This method should be `private MetaDTO createSuccessMetaData(){` as rest anytype of Meta Data will be handled by exception handler as per logic